src/bot/utils.ts

```diff
@@ -211,3 +214,85 @@ export async function paginate(
 
   return paginate(interaction, embed, pages, page, message);
 }
+
+export async function waitForConfirmation(
+  interaction: ChatInputCommandInteraction,
+  embed: APIEmbed,
+  sendType: string
+): Promise<ButtonInteraction | null> {
+  const message = await interaction.reply({
+    content: `Êtes-vous sûr de vouloir confirmer la proposition de ce${
+      sendType === 'report' ? ' signalement' : 'tte blague'
+    } ?`,
```

Comment:
Plus propre comme ça
```suggestion
    content: `Êtes-vous sûr de vouloir confirmer la proposition de ${
      sendType === 'report' ? 'ce signalement' : 'cette blague'
    } ?`,
```